react/AddressForm.tsx

```diff
@@ -1,46 +1,162 @@
-import React from 'react'
-import { Address } from 'vtex.checkout-graphql'
-import { Input } from 'vtex.styleguide'
+import React, { useState } from 'react'
+import { Input, Dropdown, ButtonPlain } from 'vtex.styleguide'
 import { useAddressContext } from 'vtex.address-context/AddressContext'
-import { LineFragment } from './typings/countryRulesTypes'
+import {
+  LineFragment,
+  Fields,
+  OptionsField,
+  Display,
+} from './typings/countryRulesTypes.d'
+import rules from './countries/rules'
+import PlaceDetails from './PlaceDetails'
+import { FormattedMessage, defineMessages } from 'react-intl'
+import NumberOption from './components/NumberOption'
 
-interface Props {
-  address: Address
-  summary: LineFragment[][]
-}
+defineMessages({
```

Comment:
shouldn't you use the return value of this function?

Reply:
We reviewed this together, please confirm it's okay now :+1: